File: omdev/cc/tests/src/hi.cc
//usr/bin/true; exec om cc run "$0" "$@"
#include <iostream>

int main(int argc, const char *argv[]) {
   std::cout << "Arguments (" << argc << "):" << std::endl;

  for (int i = 0; i < argc; ++i) {
    std::cout << "argv[" << i << "]: " << argv[i] << std::endl;
  }

  return 0;
}

File: omdev/java/tests/src/Hi.java
//usr/bin/true; exec om java run "$0" "$@"
public class Hi {
    public static void main(String[] args) {
        System.out.printf("Arguments (%d):\n", args.length);
        for (int i = 0; i < args.length; ++i) {
            System.out.printf("argv[%d]: %s\n", i, args[i]);
        }
    }
}
